spec/factories/tasks.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :task do
+    title       { Faker::Lorem.sentence(word_count: 2) }
+    description { Faker::Lorem.sentence(word_count: 10) }
+    association :user, factory: :user
```

Comment:
どちらでも大丈夫です。 こんな感じの書き方もいけます 👍 

```suggestion
    user { create(:user) }
```

Reply:
https://github.com/cat2koban/kalendar/pull/42/commits/1e88444496ce735dfeb53cfec621fb7381195539 にて対応いたしました。